src/components/templates/PartyMap/components/RoomList/RoomList.tsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
-import "./RoomList.scss";
 
-import { RoomData } from "types/RoomData";
-import RoomCard from "../RoomCard";
 import { eventHappeningNow } from "utils/time";
+import { RoomData } from "types/RoomData";
+
+import { RoomCard } from "..";
```

Comment:
.

Reply:
Will resolve this one as above, but will leave the above open for future reference.